src/main/java/TextView.java

```diff
@@ -20,9 +20,13 @@
 	public void update(Observable o, Object arg)
 	{
 		Model model = (Model) o;
+		
```

Comment:
- [ ] General Board Display
    - [ ] The board spacing seems a little off. Each column is a little to the right of the column label.
    - [ ] The top and bottom boundaries of the grid do not extend fully to the right.